tools/ci_build/github/apple/default_mobile_ios_framework_build_settings.json

```diff
@@ -8,19 +8,28 @@
             "x86_64"
         ]
     },
-    "build_params": [
-        "--ios",
-        "--parallel",
-        "--use_xcode",
-        "--build_apple_framework",
-        "--minimal_build=extended",
-        "--disable_rtti",
-        "--disable_ml_ops",
-        "--disable_exceptions",
-        "--enable_reduced_operator_type_support",
```

Comment:
Is it intentional to remove the minimal build related flags from this config?

Reply:
oops, was populating from the full build json one... fixed.